javascripts/govuk/shim-links-with-button-role.js

```diff
@@ -1,13 +1,11 @@
 // javascript 'shim' to trigger the click event of element(s)
 // when the space key is pressed.
 //
-// usage instructions:
-// GOVUK.shimLinksWithButtonRole.init();
+// Created since some Assistive Technologies (for example some Screenreaders)
```

Comment:
Here's a video of our test on buttons vs links: https://www.youtube.com/watch?v=NO7u2tEbFQY